nautobot/core/templates/generic/object_edit.html

```diff
@@ -56,7 +56,4 @@ <h3>
             </div>
         </div>
     </form>
-    {% if obj and settings.DOCS_ROOT %}
```

Comment:
Does this set of changes make it so that `DOCS_ROOT` is no longer meaningful as a setting? If so, can we remove it from the codebase and documentation entirely?

Reply:
Yes it does, I meant to add that to the checklist of things to do before marking this PR as ready. I will remove DOCS_ROOT.